.github/CODEOWNERS

```diff
@@ -70,7 +70,7 @@ yarn.lock                                                               @backsta
 /workspaces/linkerd                                                     @backstage/community-plugins-maintainers
 /workspaces/manage                                                      @backstage/community-plugins-maintainers  @grantila
 /workspaces/matomo                                                      @backstage/community-plugins-maintainers  @yashoswalyo @deshmukhmayur @riginoommen
-/workspaces/mend                                                        @backstage/community-plugins-maintainers  @dariuszsobkowicz
+/workspaces/mend                                                        @backstage/community-plugins-maintainers  @NormanWenzelWSS @rupalvihol @hetsaliya-crestdata
```

Comment:
Make sure that @NormanWenzelWSS @rupalvihol also submit and issue like you did - https://github.com/backstage/community/issues/222. They each need to be submitted individually. This isn't a blocking comment, just wanting to make sure that doesn't get missed.